cli/src/rich/mod.rs

```diff
@@ -361,9 +456,137 @@ fn draw_input(screen: &mut dyn Write, input: &Input) -> io::Result<()> {
     Ok(())
 }
 
+fn draw_autocomplete(screen: &mut dyn Write, autocomplete: Option<&Autocomplete>) -> io::Result<()> {
+    let ac = match autocomplete {
+        Some(ac) => ac,
+        None => return Ok(()),
+    };
+
+    let max_term_width = match ac.suggestions.iter().map(|i| i.term.len()).max() {
+        Some(size) => size,
+        None => return Ok(()),
+    };
+    let max_summary_width = match ac.suggestions.iter().map(|i| i.summary.len()).max() {
+        Some(size) => size,
+        None => return Ok(()),
+    };
+    let width = max_term_width + max_summary_width + 2;
+
+    let (_, term_height) = termion::terminal_size().unwrap();
+    let start_row = term_height - 2 - ac.len();
+
+    write!(
+        screen,
+        "{}{}",
+        termion::color::Fg(termion::color::Black),
+        termion::color::Bg(termion::color::LightBlack),
+    )?;
+
+    for (pos, suggestion) in ac.suggestions.iter().enumerate() {
+        let offset: u16 = pos.try_into().unwrap();
+
+        write!(
+            screen,
+            "{}",
+            termion::cursor::Goto(3, start_row + offset),
+        )?;
+
+        if Some(pos) == ac.index {
+            write!(
+                screen,
+                "{}{}",
+                termion::style::Italic,
+                termion::color::Fg(termion::color::White),
+            )?;
+        }
+
+        write!(screen, "{}", suggestion.term)?;
+        for _ in 0..(width - suggestion.term.len() - suggestion.summary.len()) {
+            write!(screen, "{}", " ")?;
+        }
+        write!(screen, "{}", suggestion.summary)?;
```

Comment:
The web autocomplete has 1 space padding on either side, which looks a little nicer with the selection bar.

```suggestion
        write!(screen, " {}", suggestion.term)?;
        for _ in 0..(width - suggestion.term.len() - suggestion.summary.len()) {
            write!(screen, " ")?;
        }
        write!(screen, "{} ", suggestion.summary)?;
```

Reply:
Ah yes much better.